src/basic_properties.jl

```diff
@@ -89,3 +89,25 @@ function voigt_velocities(vpv, vsv, vph, vsh, η)
     vs = sqrt(μ)
     vp, vs
 end
+
+"""
+    discontinuities(m::LinearLayeredModel; depths=false)
+
+Return the radii (or depths if `depths=true`) of velocity discontinuities
+for the 1D `LinearLayeredModel` `m`.
+
+"""
+function discontinuities(m::LinearLayeredModel; depths=false)
+    @assert issorted(m.r)
```

Comment:
The radii are checked for being sorted (not as elegantly as this) in the constructors, so this shouldn't be needed unless someone has been messing with the internals of the model.